example/scripts/package-pipenv-deps.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+DIR=$1
+# find packages location
+echo "Entering $DIR"
+cd "$DIR" || exit 1
+# ensure packages are up to date
+export PIPENV_VENV_IN_PROJECT=1
+pip install pipenv
+pipenv install --deploy --ignore-pipfile
```

Comment:
There's a difference between this command the `build` task for `python-reference-task`, which includes a `--dev` flag. Is that a problem?

Reply:
The only difference is whether it will install `pylint`, which we don't need in the lambda package.